sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -111,4 +111,18 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
   }
 }
 
-class ShowTablesSuite extends ShowTablesSuiteBase with SharedSparkSession
+class ShowTablesSuite extends ShowTablesSuiteBase with SharedSparkSession {
+  test("show partitions from a datasource table") {
```

Comment:
Shall we add a JIRA prefix, @MaxGekk?

Reply:
I didn't add because tests from the suite have a prefix already. See the base trait where we override the test() method. Maybe, 2 prefixes are ok.